authentication/views.py

```diff
@@ -246,6 +246,9 @@ def get(self, request):
 
     def post(self, request):
         data = request.data
+        serializer = self.serializer_class(data = request.data)
+        serializer.is_valid(raise_exception=True)
+        serializer.save()
```

Comment:
@shemaeric Why are we calling `serializer.save()` here, since we have it on line **271**?